### ISSUE TITLE
3d models not loading in mobile view

### ISSUE THREAD

**Issue by MontilvaJ2201** [adrianhajdin/project_3D_developer_portfolio#109]
Hello, I followed all the steps in the YouTube video, when I try to display the portfolio using Vercel it seems to work correctly when I view it from a computer, but when I enter the portfolio from my cell phone the 3D models of the computer, the balls, the stars and the planet are not loading correctly. In the hero where the computer is directly, the page goes blank, I continue down and when I get to the balls, most of them do not load, the rest seem to load normally but after a while they fall. And finally when I get to the contact section where the planet and the stars are, sometimes it loads perfectly but if I restart the page or after a while they crash and the page remains blank, only the form is visible. I have been trying to solve the error for some time but I no longer have a clue.

**Comment by ilyasTraikia** [comment]
same here , so did u find any solution yet ?

**Comment by sheharyarahmad842** [comment]
Yes I am also facing the same issue. I thought it was some issue in production but I have deployed on both Netlify and Render but the issue remains same.

**Comment by responsive-we** [comment]
Same here Have you found the solutions.

**Comment by responsive-we** [comment]
Yes remove the 3d balls for skills. It will work properly.
The reason behind that is mobile browsers limit the amount of 3d canvas you can load. So removing the balls will work for u.
Thank you.

**Comment by naushad1308** [comment]
> Yes remove the 3d balls for skills. It will work properly. The reason behind that is mobile browsers limit the amount of 3d canvas you can load. So removing the balls will work for u. Thank you.

Thankyou
